File: GuessingGame.py
#we will be creating a guessing game in pythonn
answer_ = "Sending"
guess_ = ""

num_guesses_ = 0
max_guesses_ = 3
is_out_of_guesses_ = False
while guess_ != answer_ and is_out_of_guesses_ == False:
    if num_guesses_ < max_guesses_:
        guess_ = input("Enter guess:")
        num_guesses_ += 1
    else:
        is_out_of_guesses_ = True




if guess_ == answer_ or num_guesses_ < max_guesses_:
    print("You win")
else:
    print("You lose")

File: GuessingGame.cpp
#include <iostream>

using namespace std;















int main() //eqiuivalent to public static void main(String[] args)
{

    int secretNum = 7;
    int guess;
    int guessCount = 0;
    int guessLimit = 3;
    bool outOfGuesses = false;

    while(secretNum != guess && outOfGuesses = false)
    {
        
        if(guessCount < guessLimit)
        {
            cout <<"Enter guess: ";
            cin >> guess;
            guessCount++;
        }
        else
        {
            outOfGuesses = true;
        }
        
        
    }
    if(outOfGuesses == true)
    {
            cout << "You Lose!";

    }
    else
    {
             cout << "You Win!";
    }

    return 0;
}